indico/web/assets/vars_js.py

```diff
@@ -205,7 +209,8 @@ def generate_global_file():
         'Data': {
             'WeekDays': ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'],
             'DefaultLocation': default_location,
-            'Locations': location_names
+            'Locations': location_names,
+            'RoomsSpriteToken': _cache.get('rooms-sprite-token'),
```

Comment:
this is not a good place for it. that file has extremely strong caching (indico version + indico.conf hash) so an updated token is not likely to show up in here anytime soon.